README.md

```diff
@@ -77,6 +77,48 @@ You can now start your server!
 $ rails s
 ```
 
+### (Optional) Social providers integration
```

Comment:
No that I want to troll you, but I'd put this into docs xD. We can do it later and also create an index / GitHub page.